.storybook/preview-head.html

```diff
@@ -0,0 +1,46 @@
+<style>
```

Comment:
Note: if it where not for this improvement (the added fonts), the diff size would have more deletions! --> ✌️